lib/common/acl.c

```diff
@@ -154,6 +154,7 @@ __xml_acl_create(xmlNode *xml, GList *acls, enum xml_private_flags mode)
  *
  * \return New head of (possibly modified) acls
  */
+PCMK__XML_DIRECTLY_RECURSIVE
```

Comment:
This requires someone reading to code to go grep for this constant. I think it makes more sense to just add "\note This function is recursive" in the doxygen block, where it will be useful for API readers as well.